### ISSUE TITLE
cannot return equivalents of null from PythonTask

### ISSUE THREAD

**Issue by samthiriot** [openmole/openmole#496]
Dear beloved OpenMole community,

I tried to implement using a PythonTask a function which sometimes should return the fact the value cannot be computed. 
In Scala I return null or NaN. 

In Python returning None lead to a json error when OpenMole tries to unpack the value from Python to Scala, something like "unable to unpack JNull"

In Python returning nan lead to the following error:
```
Caused by: com.fasterxml.jackson.core.JsonParseException: Non-standard token 'NaN': enable JsonParser.Feature.ALLOW_NON_NUMERIC_NUMBERS to allow
```

It's a bit sad not being able to return these specific values from Python.

Sam

**Comment by romainreuillon** [comment]
Dear Sam,

I see two parts here. 

The first one is to be able properly transcribe the NaN value in output of a Python/R Task. This concern only output of type Val[Double].

The second part would be to support None value in Python. I think it should be mapped to Val[Option[...]] in OpenMOLE. Do you aggree ?

Cheers,
Romain

**Comment by samthiriot** [comment]
Hello Romain,
More generally for numeric types (integer as well), there are these specific values of: NaN: positive infinity, negative infinity. 

You're true, None is probably different. I understand it as "null" is Java, C or R. I'm not fluent enough in Scala nor OpenMOLE to know what a good translation would be. 

Actually from the point of view of the user, I believe what matters is to be able to return "I've no results for that"; if it can be done with "NaN" then this funcitonal need is covered, and no need for null. 

HTH, 

Sam

**Comment by romainreuillon** [comment]
Dear Sam,

I agree that we could focus in handling NaN (and possibly infinity values) for Double produced by Python.

Could you provide a full stack for the openmole parsing error ? 

Romain

**Comment by romainreuillon** [comment]
Dear Sam,

This is fixed in 8c81d2200efb5e70460f7bf3331afbe626c63490

(FYI it is being integrated in the 13.0-SNAPSHOT docker image by the CI https://gitlab.openmole.org/openmole/openmole/-/pipelines)

Cheers,
Romain
